### PR TITLE
✨ Add version update command

### PR DESCRIPTION
**Pull request by fiendish**

### CODE REVIEW

File: ncpi_fhir_utility/update_versions.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+def run(site_root, new_version, our_urls):
+    our_urls = tuple(s.strip() for s in our_urls.split(","))
+    site_root = Path(site_root)
+    if not site_root.is_dir():
+        print(
+            f"Error: <site_root> dir '{site_root}' must exist.", file=sys.stderr
+        )
+    ini_file = Path(site_root, "ig.ini")
+    if not ini_file.is_file():
+        print(f"Error: <ini_file> '{ini_file}' must exist.", file=sys.stderr)
```
Comment (znatty22) [inline ncpi_fhir_utility/update_versions.py:51]: I think this should be able to use the logger?

File: ncpi_fhir_utility/update_versions.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+"""
+import json
+import sys
+from datetime import datetime, timezone
+from pathlib import Path
+
+import click
+
+
+def bump_ig_version(filePath, new_version):
```
Comment (znatty22) [inline ncpi_fhir_utility/update_versions.py:16]: I think you can use Python's ConfigParser to read `.ini` files? https://docs.python.org/3.7/library/configparser.html

File: ncpi_fhir_utility/update_versions.py (hunk 1 above)
Comment (fiendish) [reply ncpi_fhir_utility/update_versions.py:51]: Yes but why?

File: ncpi_fhir_utility/update_versions.py (hunk 2 above)
Comment (fiendish) [reply ncpi_fhir_utility/update_versions.py:17]: √

File: ncpi_fhir_utility/update_versions.py (hunk 1 above)
Comment (fiendish) [reply ncpi_fhir_utility/update_versions.py]: √

File: ncpi_fhir_utility/update_versions.py [hunk 3]
```diff
@@ -14,15 +15,14 @@
...
-            line
-            if not line.startswith("version ")
-            else f"version = {new_version}\n"
-            for line in f
-        ]
-    with filePath.open(mode="w") as f:
-        f.writelines(body)
+    config = configparser.ConfigParser()
+    if config.read(filePath) and "IG" in config:
+        if "version" in config["IG"]:
```
Comment (znatty22) [inline ncpi_fhir_utility/update_versions.py:20]: If a version is not in the `ini` it probably should be?

Comment (znatty22) [reply ncpi_fhir_utility/update_versions.py:20]: I think same goes with `date` in the resource payload. If it's not there, we should probably add it.

Comment (fiendish) [reply ncpi_fhir_utility/update_versions.py:20]: no idea! but we can certainly do that

**Comment by fiendish** [comment]
> I tried this on `kf-model-fhir` and saw that none of the resources ended up with a `version`

??? I see version field changes on all of these files:

```
M site_root/ig.ini
M site_root/input/ImplementationGuide-KidsFirst.json
M site_root/input/resources/extensions/StructureDefinition-age-at-event.json
M site_root/input/resources/extensions/StructureDefinition-concentration.json
M site_root/input/resources/extensions/StructureDefinition-display-name.json
M site_root/input/resources/extensions/StructureDefinition-related-organization.json
M site_root/input/resources/extensions/StructureDefinition-relation.json
M site_root/input/resources/extensions/StructureDefinition-species.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-condition-no-phi.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-condition.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-patient-no-phi.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-patient.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-phenotype-no-phi.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-phenotype.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-research-study.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-specimen-no-phi.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-specimen.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-vital-status-no-phi.json
M site_root/input/resources/profiles/StructureDefinition-kfdrc-vital-status.json
M site_root/input/resources/search/SearchParameter-age-at-event.json
M site_root/input/resources/search/SearchParameter-concentration.json
M site_root/input/resources/search/SearchParameter-display-name.json
M site_root/input/resources/search/SearchParameter-related-organization-role.json
M site_root/input/resources/search/SearchParameter-related-organization.json
M site_root/input/resources/search/SearchParameter-relation-note.json
M site_root/input/resources/search/SearchParameter-relation-subject.json
M site_root/input/resources/search/SearchParameter-relation.json
M site_root/input/resources/search/SearchParameter-species.json
M site_root/input/resources/terminology/CodeSystem-species.json
M site_root/input/resources/terminology/ValueSet-phenotype-codes.json
M site_root/input/resources/terminology/ValueSet-species.json
M site_root/package.json
```

> or a `date`

That's true, but I need more inofrmation about what you'r talking about re: version.

**Comment by fiendish** [comment]
If they don't have a url, how do we know that they're ours?

**Comment by znatty22** [comment]
@fiendish > If they don't have a url, how should we detect that they're ours?

Hmm for the non-conformance resources, you could check the list of `meta.profiles` to see if any of the URLs start with our urls? 

But regardless of ^ I think we should decide first if we want example resources to be part of our versioned model or not? If we say they aren't part of the FHIR model then its probably ok to not version them, which means what you have now works. If they are part of the FHIR model we still need to figure out where to store the version in the resource body since it's diff between example resources and conformance resources.

**Pull request merged**